lib/arena.rb

```diff
@@ -1,2 +1,76 @@
+require_relative 'gladiator'
+
 class Arena
+  attr_accessor :name, :gladiators, :audience_response
+
+  def initialize(name)
+    @name = name.capitalize
+    @gladiators = []
+    @audience_response = ""
+  end
+
+  def add_gladiator(gladiator)
+    if @gladiators.length < 2
+       @gladiators.push(gladiator)
+    end
+  end
+
+  def fight
+
+    weapons_array = ["Sword", "Club", "Trident", "Spear"]
+    rand_weapons_array = weapons_array.shuffle
+
+    maximus = Gladiator.new("Maximus", rand_weapons_array[0])
+    ephates = Gladiator.new("Ephates", rand_weapons_array[1])
+    bilcephalon = Gladiator.new("Bilcephalon", rand_weapons_array[2])
+
+    gladiator_array = [maximus, ephates, bilcephalon]
+    rand_gladiator_array = gladiator_array.shuffle
+    add_rand_glad = add_gladiator(rand_gladiator_array[0])
+
+
+    puts "A random gladiator with a random weapon will battle your gladiator of choice? Choose M for Maximus, E for Ephates or B for Bilcephalon?"
+    @audience_response = gets.chomp
+    @audience_response.downcase!
+
+    if @audience_response == "m"
+      add_gladiator(maximus)
+      rand_gladiator_array -= [maximus]
+      add_rand_glad
```

Comment:
What exactly is `add_rand_glad` doing at the end of each conditional here? It contains a static value (some random gladiator from the rand_gladiator_array) -- it's not calling a method.